sdk/resourcemanager/azure-resourcemanager-sql/src/main/java/com/azure/resourcemanager/sql/models/SqlDatabase.java

```diff
@@ -702,11 +703,9 @@
             /**
              * Sets the edition for the SQL Database.
              *
-             * @deprecated use {@link WithEditionDefaults}
              * @param edition edition to be set for database
              * @return The next stage of the definition
              */
-            @Deprecated
             SqlDatabase.DefinitionStages.WithAttachAllOptions<ParentT> withEdition(DatabaseEdition edition);
```

Comment:
I assume the change will be remove this stage, and keep `WithEditionDefaults` stage, assume the `@deprecated` doc is correct?